automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/summary/WorkspaceSummaryPage.scala

```diff
@@ -279,6 +279,11 @@ class WorkspaceSummaryPage(namespace: String, name: String)(implicit webDriver:
     else
       throw new IllegalStateException("Already editing")
 
+    val start = System.currentTimeMillis()
```

Comment:
What is this section doing?

Reply:
this section mashed the edit button until it works

Reply:
I don't like this, and I found an explanation that has been discussed in #dsde-qa having to do with clicking the edit button immediately when it's enabled and the page briefly switching to edit mode and then back to view mode. Matt Putnam is looking into it.